ppr-ui/src/composables/mhrRegistration/useMhrValidations.ts

```diff
@@ -23,24 +28,20 @@ export const useMhrValidations = (validationState: any) => {
 
   /** Reset submission validations to default . */
   const resetAllValidations = (): void => {
-    // Reset your home validations
-    setValidation(MhrSectVal.YOUR_HOME_VALID, MhrCompVal.MAKE_MODEL_VALID, false)
-    setValidation(MhrSectVal.YOUR_HOME_VALID, MhrCompVal.HOME_SECTION_VALID, false)
-    setValidation(MhrSectVal.YOUR_HOME_VALID, MhrCompVal.HOME_CERTIFICATION_VALID, false)
-    setValidation(MhrSectVal.YOUR_HOME_VALID, MhrCompVal.REBUILT_STATUS_VALID, false)
-    setValidation(MhrSectVal.YOUR_HOME_VALID, MhrCompVal.OTHER_VALID, false)
-    // Reset submitting party validations
-    setValidation(MhrSectVal.SUBMITTING_PARTY_VALID, MhrCompVal.SUBMITTER_VALID, false)
-    setValidation(MhrSectVal.SUBMITTING_PARTY_VALID, MhrCompVal.DOC_ID_VALID, false)
-    setValidation(MhrSectVal.SUBMITTING_PARTY_VALID, MhrCompVal.REF_NUM_VALID, false)
-    // Reset home owner validations
-    setValidation(MhrSectVal.HOME_OWNERS_VALID, MhrCompVal.OWNERS_VALID, false)
-    // Reset home location validations
-    setValidation(MhrSectVal.LOCATION_VALID, MhrCompVal.LOCATION_TYPE_VALID, false)
-    setValidation(MhrSectVal.LOCATION_VALID, MhrCompVal.CIVIC_ADDRESS_VALID, false)
-
-    // Reset Review validations
-    setValidation(MhrSectVal.REVIEW_CONFIRM_VALID, MhrCompVal.VALIDATE_STEPS, false)
+    // Gets all sections in model
+    const sections = Object.keys(validationState) as MhrSectVal[]
+    for (const section of sections) {
+      // Gets all components for the specifc section
+      const components = Object.keys(validationState[section].value) as MhrCompVal[]
+      // Sets all components to false
+      for (const component of components) {
+        setValidation(section, component, false)
+      }
+    }
+    // Default for ADD_EDIT_OWNERS_VALID -> OWNERS_VALID is true
+    if (sections.includes(MhrSectVal.ADD_EDIT_OWNERS_VALID)) {
+      setValidation(MhrSectVal.ADD_EDIT_OWNERS_VALID, MhrCompVal.OWNERS_VALID, true)
+    }
```

Comment:
This resets all validations to their default values based on the model:

![image](https://github.com/bcgov/ppr/assets/77707952/067326eb-4373-43f9-987b-834a024d8748)

Reply:
`addEditOwnersvalid` doesn't really follow the 'steps` model per se but i'm assuming this was pre-existing. 
A refactor for another day. 
Should* probably be part of the homeOwnersValid block.

Reply:
Agreed I thought the same!